testnets/neuradevnet/assetlist.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "neuradevnet",
+  "assets": [
+    {
+      "description": "ANKR: The native EVM, governance, and staking token for Neura, enabling secure transactions, and seamless GPU resourcing within the ecosystem.",
+      "denom_units": [
+        {
+            "denom": "atankr",
+            "exponent": 0
+        },
+        {
+            "denom": "ankr",
+            "exponent": 18
+        }
+      ],
+      "base": "atankr",
+      "name": "Neura Devnet",
+      "display": "ankr",
+      "symbol": "ANKR",
+      "traces": [
+        {
+          "type": "test-mintage",
+          "counterparty": {
+            "chain_name": "neura",
+            "base_denom": "atankr"
+          },
+          "provider": "Neura"
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "neura"
+          },
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neura/images/ankr.svg"
```

Comment:
Just letting you know that Sync Images Action will automatically replace this reference to ankr.svg with neura.png, because it's pointing to neura's chain image, which is defined as neura.png (not ankr.svg).

Reply:
Do I need to remove the `image_sync ` then?

Reply:
actually it should be pointing to the asset
```
  "image_sync": {
    "chain_name": "neura",
    "base_denom": "atankr"
  },
```